### PR TITLE
[CodeCamp#158] SimCC 1d heatmap visualization

### PR DESCRIPTION
**Pull request by jack0rich**
## Motivation
Visualize SimCC one-dimensional heatmap.

## Modification
A new method named `_draw_instance_xy_heap` is extended in `PoseLocalVisualizer` to support the visualization of heatmaps such as SimCC.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by ly015** [comment]
Thank you very much for your contribution. This PR will be reviewed asap.

File: mmpose/visualization/local_visualizer.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 from mmpose.registry import VISUALIZERS
 from mmpose.structures import PoseDataSample
+from simCCVis import SimCCVisualizer
```
Comment (Tau-J) [inline mmpose/visualization/local_visualizer.py:15]: I suggest renaming this file to `simcc_vis.py` and place it under `mmpose/visualization` folder

**Comment by Tau-J** [comment]
Hi @jack0rich , would you mind fixing the lint and adding an example of simcc visualization result to [visualization docs](https://mmpose.readthedocs.io/en/1.x/user_guides/visualization.html)?

**Comment by jack0rich** [comment]
> Hi @jack0rich , would you mind fixing the lint and adding an example of simcc visualization result to [visualization docs](https://mmpose.readthedocs.io/en/1.x/user_guides/visualization.html)?

Sure, but I'm confused about lint.Is there a problem with the copyright of simcc_vis.py?
93F2344E-E7F8-42FF-8352-E0BAFD69FBE5

**Comment by Ben-Louis** [comment]
> Sure, but I'm confused about lint.Is there a problem with the copyright of simcc_vis.py? 93F2344E-E7F8-42FF-8352-E0BAFD69FBE5

Hi @jack0rich, the copyright description 
```
# Copyright (c) OpenMMLab. All rights reserved.
```
is needed at the top of new files

**Review by Tau-J — APPROVED** [review]

**Pull request merged**